frontend/src/components/Calendar/CalendarHeader.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import styles from "./CalendarHeader.module.css";
+import icon from "../../assets/images/addicon.png";
+
+const CalendarHeader = () => {
+    return (
+        <header className={`d-flex align-items-center ${styles.header}`}>
+            <div className={styles.buttonContainer}>
+                <div className={`${styles.icon_sentence} d-flex align-items-center mt-4`}>
+                    <img src={icon} alt="Add Icon" className={styles.icon} />
+                    <span className={styles.sentence}>Create</span>
+                </div>
+                <button className={styles.button}>Today</button>

```

Comment:
- It will be really cool if we can add the function for user to switch/select different date/month!

Reply:
Agree but I have no idea how to do that yet, later ><